ci/cloudfront-orchestration/monitoring/staging/parameters.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "ParameterKey": "CloudfrontDistribution",
```

Comment:
```suggestion
    "ParameterKey": "CloudFrontDistribution",
```

Reply:
Nitpick, feel free to resolve when you want to merge

Reply:
This has to match https://github.com/govuk-one-login/devplatform-deploy/tree/main/cloudfront-monitoring-alarm#:~:text=CloudfrontDistribution

Reply:
I had a feeling it might be something like that :(